evals/agent_evals/test_cases.json

```diff
@@ -0,0 +1,115 @@
+[
+  {
+    "test_case_name": "test_case_1",
```

Comment:
This is a pretty cool json format. It seems like this will be really easy to define new tests and also get a quick understanding of what the tests are supposed to do